src/ParallelAlgorithms/Initialization/Actions/RemoveOptionsAndTerminatePhase.hpp

```diff
@@ -33,7 +33,14 @@ struct RemoveOptionsAndTerminatePhase {
                     const Parallel::ConstGlobalCache<Metavariables>& /*cache*/,
                     const ArrayIndex& /*array_index*/, ActionList /*meta*/,
                     const ParallelComponent* const /*meta*/) noexcept {
-    using tags_to_remove = Parallel::get_initialization_tags<ActionList>;
+    using initialization_tags = Parallel::get_initialization_tags<ActionList>;
+    using initialization_tags_to_keep =
+        Parallel::get_initialization_tags_to_keep<ActionList>;
+    // Keep the tags that are in initialization_tags_to_keep.
+    using tags_to_remove = tmpl::remove_if<
```

Comment:
is it possible to use `tmpl::list_difference<initialization_tags, initialization_tags_to_keep>`?